lib/ubus.c

```diff
@@ -572,6 +572,40 @@ uc_ubus_call_cb(struct ubus_request *req, int type, struct blob_attr *msg)
 	}
 }
 
+static void
+uc_ubus_vm_handle_exception(uc_vm_t *vm)
+{
+	uc_value_t *exh, *val;
+
+	exh = uc_vm_registry_get(vm, "ubus.ex_handler");
+	if (!ucv_is_callable(exh))
+		goto error;
+
+	val = uc_vm_exception_value(vm);
+	uc_vm_stack_push(vm, ucv_get(exh));
+	uc_vm_stack_push(vm, ucv_get(val));
+
+	if (uc_vm_call(vm, false, 1) != EXCEPTION_NONE)
```

Comment:
We're leaking `val` here. Best is to push it onto the stack without `ucv_get()` to transfer ownership entirely to the VM stack, so the funciton call will take care of freeing it.